src/components/StartScriptGenerator.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { useEffect, useState, useRef } from 'react';
+import '@site/src/css/start-script-generator.css';
+
+const markerPoints = [4, 8, 12, 16, 20];
+
+type FlagType = {
+    label: string;
+    value: string;
+    description: string;
+};
+
+const WINDOWS_AUTO_RESTART = ":start\n%%CONTENT%%\n\necho Server restarting...\necho Press CTRL + C to stop.\ngoto :start";
+const LINUX_AUTO_RESTART = "while [ true ]; do\n    %%CONTENT%%\n    echo Server restarting...\n    echo Press CTRL + C to stop.\ndone";
+
+const FLAGS: { [key: string]: FlagType } = {
+    AIKARS: {
+        label: 'Aikar\'s',
+        value: '-XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1 -Dusing.aikars.flags=https://mcflags.emc.gs -Daikars.new.flags=true -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20',
```

Comment:
Can we do something like `['-XX:+UseG1GC', '-XX:+ParallelRefProcEnabled', ..].join(' ')`, and keep each on their own line?

Reply:
Yeah, that does increase the readability and makes it easier to modify them in the future 😄

Reply:
Can you do it for the others too?